### PR TITLE
MIXEDARCH-356: bring intel control plane 4.14 job to 4.15 periodics

### PR DESCRIPTION
**Pull request by pkenchap**
Adding the 4.15 Jobs for Intel control plane .

### CODE REVIEW

**Comment by pkenchap** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-ibmcloud-ovn-multi-ppc64le

File: ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.14.yaml [hunk 1]
```diff
@@ -318,7 +318,7 @@ tests:
       FEATURE_SET: TechPreviewNoUpgrade
     workflow: openshift-e2e-aws-serial
 - as: ocp-e2e-ibmcloud-ovn-multi-ppc64le
-  cron: 20 11,23 * * *
+  cron: 20 19 * * *
```
Comment (prb112) [inline ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.14.yaml:321]:
```suggestion
  cron: 20 23 * * *
```

File: ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.15.yaml [hunk 2]
```diff
@@ -317,6 +317,40 @@ tests:
       COMPUTE_NODE_TYPE: m6g.xlarge
       FEATURE_SET: TechPreviewNoUpgrade
     workflow: openshift-e2e-aws-serial
+- as: ocp-e2e-ibmcloud-ovn-multi-ppc64le
+  cron: 20 7,19 * * *
```
Comment (prb112) [inline ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.15.yaml:321]:
```suggestion
  cron: 20 7,17 * * *
```

File: ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.14.yaml (hunk 1 above)
Comment (prb112) [reply ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.14.yaml:321]: I'm thinking we leave this one at 23

File: ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.15.yaml (hunk 2 above)
Comment (prb112) [reply ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.15.yaml:321]: Let's not use the same exact execution hour as the 4.14 job, let's use 2 hrs earlier

File: ci-operator/jobs/openshift/multiarch/openshift-multiarch-master-periodics.yaml [hunk 3]
```diff
@@ -11154,7 +11154,7 @@ periodics:
         secretName: result-aggregator
 - agent: kubernetes
   cluster: build03
-  cron: 20 11,23 * * *
+  cron: 20 19 * * *
```
Comment (prb112) [inline ci-operator/jobs/openshift/multiarch/openshift-multiarch-master-periodics.yaml:11157]:
```suggestion
  cron: 20 23 * * *
```

File: ci-operator/jobs/openshift/multiarch/openshift-multiarch-master-periodics.yaml [hunk 4]
```diff
@@ -14652,6 +14652,88 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build03
+  cron: 20 7,19 * * *
```
Comment (prb112) [inline ci-operator/jobs/openshift/multiarch/openshift-multiarch-master-periodics.yaml:14657]:
```suggestion
  cron: 20 7,17 * * *
```

**Review by prb112 — APPROVED** [review]
LGTM

**Pull request merged**